sdk/include/ddk/usbprotocoldefs.h

```diff
@@ -541,7 +541,7 @@ typedef struct _USBHUBPORTDATA {
 #define USB_LANG_TURKISH                                    0x1f
 #define USB_LANG_URDU                                       0x20
 #define USB_LANG_INDONESIAN                                 0x21
-#define USB_LANG_UKRANIAN                                   0x22
+#define USB_LANG_UKRAINIAN                                  0x22
```

Comment:
No! You are breaking compatibility with official headers:
https://github.com/Aekras1a/Labs/blob/master/Labs/WinDDK/7600.16385.1/inc/ddk/USBProtocolDefs.h#L742

Reply:
```suggestion
#define USB_LANG_UKRANIAN                                   0x22
```